openapi-cli/src/main/resources/templates/client/client-ep.mustache

```diff
@@ -10,46 +10,61 @@ public client class {{cut info.title " "}}Client {
     public {{cut info.title " "}}ClientConfig config;
 
     public function __init({{cut info.title " "}}ClientConfig config) {
-        http:Client httpEp = new(config.serviceUrl, {auth: config.clientConfig.auth, cache: config.clientConfig.cache});
+        http:Client httpEp = checkpanic new(config.serviceUrl, {auth: config.clientConfig.auth, cache:
+        config.clientConfig.cache});
         self.clientEp = httpEp;
         self.config = config;
     }
 {{#paths}}{{#value}}{{#operations}}{{#value}}{{#equals key "multi"}}{{#if methods}}{{#methods}}
-    public remote function {{lower .}}{{{operationId}}}({{#pathParameters}}{{>pathParams}}{{#unless @last}}, {{/unless}}{{/pathParameters}}) returns http:Response | error {
+    remote function {{lower .}}{{{operationId}}}({{#pathParameters}}{{>pathParams}}{{#unless @last}}, {{/unless}}{{/pathParameters}}) returns http:Response | error {
         http:Client ep = self.clientEp;
         http:Request request = new;
 
         // TODO: Update the request as needed
-    {{#equals . "GET"}}return check ep->{{lower .}}("{{../../../key}}", message = request);{{else}}{{#equals . "HEAD"}}
-        return check ep->{{lower .}}({{../../../key}}, message = request);{{else}}{{#equals . "OPTIONS"}}
-        return check ep->{{lower .}}({{../../../key}}, message = request);{{/equals}}{{/equals}}
-        return check ep->{{lower .}}({{../../../key}}, request);{{/equals}}
+    {{#equals . "GET"}}var response = ep->{{lower .}}("{{../../../key}}", message = request);{{else}}{{#equals . "HEAD"}}
+        var response = ep->{{lower .}}({{../../../key}}, message = request);{{else}}{{#equals . "OPTIONS"}}
+        var response = ep->{{lower .}}({{../../../key}}, message = request);{{/equals}}{{/equals}}
+        var response = ep->{{lower .}}({{../../../key}}, request);{{/equals}}
+        if (response is http:Response) {
+            return <http:Response> response;
```

Comment:
Do we need to cast with `<http:Response>` ? with type guard its already casted right ?
Also remove the gap between `<http:Response> response` if `<http:Response>` is needed.

Reply:
Check in other places.